css-three-column-layout/styles.css

```diff
@@ -0,0 +1,60 @@
+body {
+  background-color: #1996d9;
+}
+
+img {
+  max-width: 55px;
+  margin-top: 10px;
+  margin-left: 5px;
+}
+
+h3 {
+  text-align: center;
+}
+
+p {
+  font-size: 16px;
+}
+
+.container {
+  margin-left: 10%;
+  margin-right: 10%;
+  margin-top: 20px;
+}
+
+.circle-background {
+  max-width: 80px;
+  padding: 10px;
+  background-color: white;
+  border-radius: 50%;
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  color: white;
+  font-family: 'Lato', sans-serif;
+  margin-left: 20px;
+  margin-right: 20px;
```

Comment:
These properties shouldn't be applied to the `row` class, you will want to make an additional class to apply to the `row` elements you need this behavior for. If you change the `row` class like this, then all rows in your application need to work like this, which would be bad news, this behavior is too unique to apply to all situations.